packages/opentelemetry-sdk-trace-base/src/Span.ts

```diff
@@ -66,6 +67,10 @@ export class Span implements APISpan, ReadableSpan {
   readonly startTime: HrTime;
   readonly resource: Resource;
   readonly instrumentationLibrary: InstrumentationLibrary;
+
+  readonly droppedAttributesCount: SpanDroppedAttributes = {};
+  readonly droppedEventsCount: SpanDroppedEvents = {};
```

Comment:
the attribute name doesn't really match the type. `count` sounds like it should be a (postive) `number`.

Reply:
I agree, now that I read the spec again, it is clear. So we only want to report the dropped counts, and not the actual dropped attributes.

Reply:
Yes just the counts

Reply:
Also, to confirm what this line in the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/0cb45c779a0b8a78d669aaf61c1b39ffab7dc0ae/specification/trace/sdk_exporters/non-otlp.md#dropped-attributes-count) means:
`This key-value pair should only be recorded when it contains a non-zero value.`

What does "recorded" mean here? If any of the counts is zero, should `ReadableSpan.droppedAttributesCount` be `undefined`, or does it mean that it should not be exported by an exporter?

Reply:
These are actually reported as span attributes, not top level items. The keys are given by the spec and the values should be integers.